texttospeech/cloud-client/synthesize_text.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Text-To-Speech API sample application .
+
+Example usage:
+    python synthesize_text.py --text "hello" --output hello.mp3
```

Comment:
All of the snippets currently write to `output.mp3` and sample functions don't accept a parameter for output file

`--output` is not an argument

Either add support for `--output` or remove from the example usage :)

Reply:
Fixed.